packages/select/src/components/select/multiSelect.tsx

```diff
@@ -178,17 +178,17 @@ export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IM
         Utils.safeInvoke(this.props.onQueryChange, query, evt);
     };
 
-    private handlePopoverInteraction = (nextOpenState: boolean) =>
+    private handlePopoverInteraction = (_nextOpenState: boolean) =>
         // deferring to rAF to get properly updated document.activeElement
         requestAnimationFrame(() => {
-            if (this.input != null && this.input !== document.activeElement) {
-                // the input is no longer focused so we can close the popover
-                this.setState({ isOpen: false });
-            } else if (!this.props.openOnKeyDown) {
-                // open the popover when focusing the tag input
+            if (this.input != null && this.input === document.activeElement) {
+                // the input is active, we stay open regardless of `nextOpenState`
                 this.setState({ isOpen: true });
```

Comment:
hmmmm, after looking at the docs for `openOnKeyDown`, I'm realizing this might not be right https://blueprintjs.com/docs/#select/multi-select

I think if the input is still focused, we definitely don't close (so the below branch is correct).
However, in this branch, I think we only open if `!this.props.openOnKeyDown`. I assume somewhere else there's a keydown handler that opens the popover?

Maybe the original change in https://github.com/palantir/blueprint/pull/4121 was correct
(assuming the keydown handler also calls `onInteraction`)